package app.documents.core.model.exception

data object CloudPortalNotFoundException : CloudNotFoundException()
